mx.truffleruby/mx_truffleruby.py

```diff
@@ -118,17 +118,25 @@ def ruby_testdownstream_hello(args):
     mx.run(['bin/truffleruby', '-e', 'puts "Hello Ruby!"'])
 
 def ruby_testdownstream_aot(args):
-    if len(args) != 1:
-        mx.abort("Missing path to AOT image: mx ruby_testdownstream_aot <aot_bin>")
+    if len(args) > 3:
+        mx.abort("Incorrect argument count: mx ruby_testdownstream_aot <aot_bin> [<format>] [<build_type>]")
```

Comment:
I would not leak information about the build type here.
How about arguments >= 3 (or even 2 so `format` is not special) are just passed as-is to mspec?
Do we need different formats here?